Components/Image/image.css

```diff
@@ -0,0 +1,30 @@
+:root {
+    --black: #000000;
+    --medium-font-size: medium;
+    --large-font-size: large;
+    --larger-font-size: larger;
+    --bolder-font-weight: bolder;
```

Comment:
you can give font size in relative units

Reply:
Done